integration/addition_gids_test.go

```diff
@@ -49,12 +49,8 @@ func TestAdditionalGids(t *testing.T) {
 		testImage     = GetImage(BusyBox)
 		containerName = "test-container"
 	)
-	t.Logf("Pull test image %q", testImage)
-	img, err := imageService.PullImage(&runtime.ImageSpec{Image: testImage}, nil, sbConfig)
-	require.NoError(t, err)
-	defer func() {
-		assert.NoError(t, imageService.RemoveImage(&runtime.ImageSpec{Image: img}))
-	}()
```

Comment:
We no longer remove images after this change. I'm personally fine but want to highlight the behavior change just in case.

Reply:
yeah, I think that was the purpose of the PR--to simply reuse images throughout the test run rather than keep removing, re-pulling on various tests. On Windows these images can be much larger, so has more impact.